knope.toml

```diff
@@ -20,6 +22,7 @@ type = "PrepareRelease"
 [[workflows.steps]]
 type = "Command"
 command = "git commit -m \"chore: prepare release $version\""
+variables = { "$version" = "Version" }
```

Comment:
A few lines below this there is 
```
[workflows.steps.variables]
"$version" = "Version"
```

Is that redundant now?

Reply:
Seems like it’s necessary on each line